src/test/java/bio/terra/workspace/service/iam/SamServiceTest.java

```diff
@@ -309,40 +308,6 @@ void workspaceReaderIsNotPrivateResourceReader() {
     samService.deleteControlledResource(bucketResource, defaultUserRequest());
   }
 
-  // Duplicate execution during flights is handled using Sam error codes, so this verifies expected
-  // behavior in SamService.
-  @Test
-  void duplicateResourceCreateThrows() {
-    UUID workspaceId = createWorkspaceDefaultUser();
-
-    ControlledResource bucketResource = defaultBucket(workspaceId).build();
-    samService.createControlledResource(bucketResource, null, defaultUserRequest());
-
-    SamApiException exception =
-        assertThrows(
-            SamApiException.class,
-            () -> samService.createControlledResource(bucketResource, null, defaultUserRequest()));
-    assertEquals(HttpStatus.CONFLICT.value(), exception.getApiExceptionStatus());
```

Comment:
should we test that nothing happens on the second create/delete in these tests instead of removing the tests? I think that's also behavior we want and should therefore have a test for. ditto on delete test.

Reply:
Debated that. Putting them back